internal/provider/resource_aci_endpoint_security_group.go

```diff
@@ -1732,6 +1742,33 @@ func getAndSetFvESgAttributes(ctx context.Context, diags *diag.Diagnostics, clie
 					data.Shutdown = basetypes.NewStringValue(attributeValue.(string))
 				}
 			}
+			if data.Annotation.IsUnknown() {
```

Comment:
In the aci_resource_migration PR internal/provider/resource_aci_application_epg.go you can see I initialise the empty object with `*data = *getEmptyFvAEPgResourceModel()`, and then these conditionals are not needed anymore. The functionality works either way, and this is not blocking for approval but think we should make a choice.

Reply:
I'm aware of this and will remove it once your PR gets merged. I left this because the tests on CI need to pass..